packages/schema-blocks/src/core/blocks/BlockDefinition.tsx

```diff
@@ -75,7 +75,7 @@ export default class BlockDefinition extends Definition {
 			return elements[ 0 ] as ReactElement;
 		}
 
-		return createElement( Fragment, { key: props.clientId }, elements );
+		return createElement( Fragment, { key: props.clientId }, elements.map( ( element, i ) => createElement( Fragment, { key: i }, element ) ) );
```

Comment:
```suggestion
		return createElement( Fragment, { key: props.clientId }, elements.map( ( element, index ) => 
		{ 
		    element.key = index;
		    return element;
		} );
```

Reply:
this can't be done unfortunately; elements should be named components, they're not elements until after the createElement.